src/zimscraperlib/download.py

```diff
@@ -7,17 +7,33 @@
 
 import requests
 
+from . import logger
+
 WGET_BINARY = os.getenv("WGET_BINARY", "/usr/bin/wget")
 
 
-def save_file(url, fpath):
-    """ download a binary file from its URL """
-    req = requests.get(url)
-    req.raise_for_status()
-    if not fpath.parent.exists():
-        fpath.parent.mkdir(exist_ok=True)
-    with open(fpath, "wb") as fp:
-        fp.write(req.content)
+def save_file(url, fpath, timeout=30, retries=0):
```

Comment:
Could we have retries by default ? I think most use case would prefer to have retries so those who don't would set it to zero.

Reply:
Done